### PR TITLE
Version 3.2.6 - Sampling improvements

### PR DESCRIPTION
**Pull request by konraddysput**
## Version 3.2.6
- `BacktraceClient` will apply sampling only to unhandled exceptions type error.
- Fixed annotations nullable value,
- Renamed `BacktraceUnhandledException` Classifier to `UnhandledException`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Backtrace Unity Release Notes
 
+## Version 3.2.6
+- `BacktraceClient` will apply sampling only to unhandled exceptions type error.
+- Fixed annotations nullable value,
+- Renamed `BacktraceUnhandledException` Classifier to `UnhandledException`
```
Comment (sbahra) [inline CHANGELOG.md:6]: - "classifier" not "Classifier"
- Let's just call it `error` and add another classifier for the log level.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Backtrace Unity Release Notes
 
+## Version 3.2.6
+- `BacktraceClient` will apply sampling only to unhandled exceptions type error.
+- Fixed annotations nullable value,
```
Comment (sbahra) [inline CHANGELOG.md:5]: erroneous comma

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Backtrace Unity Release Notes
 
+## Version 3.2.6
+- `BacktraceClient` will apply sampling only to unhandled exceptions type error.
```
Comment (sbahra) [inline CHANGELOG.md:4]: `to errors lacking exception information.`

File: Runtime/BacktraceClient.cs [hunk 4]
```diff
@@ -20,7 +20,7 @@ public class BacktraceClient : MonoBehaviour, IBacktraceClient
     {
         public BacktraceConfiguration Configuration;
 
-        public const string VERSION = "3.2.5";
+        public const string VERSION = "3.2.6";
```
Comment (sbahra) [inline Runtime/BacktraceClient.cs:23]: FWIW: We should've done 3.3.X instead following semver.org

File: Runtime/Model/BacktraceUnhandledException.cs [hunk 5]
```diff
@@ -365,7 +365,7 @@ private BacktraceStackFrame SetDefaultStackTraceInformation(string frameString)
         /// </summary>
         private void TrySetClassifier()
         {
-            Classifier = "BacktraceUnhandledException";
+            Classifier = "UnhandledException";
```
Comment (sbahra) [inline Runtime/Model/BacktraceUnhandledException.cs:368]: `error` `warning`

File: Runtime/Model/JsonData/Annotations.cs [hunk 6]
```diff
@@ -40,7 +40,12 @@ public Annotations(Exception exception, int gameObjectDepth)
             var environmentVariables = new Dictionary<string, string>();
             foreach (DictionaryEntry variable in Environment.GetEnvironmentVariables())
             {
-                environmentVariables.Add(variable.Key.ToString(), Regex.Escape(variable.Value.ToString() ?? "NULL"));
+                if (variable.Key == null)
```
Comment (sbahra) [inline Runtime/Model/JsonData/Annotations.cs:43]: I don't see this bug fix in release notes. Also, how does one disable environment variables?

**Review by sbahra — CHANGES_REQUESTED** [review]

File: Runtime/BacktraceClient.cs (hunk 4 above)
Comment (konraddysput) [reply Runtime/BacktraceClient.cs:23]: ah yes - but now we should continue with 3.2.x I guess?

File: Runtime/Model/JsonData/Annotations.cs (hunk 6 above)
Comment (konraddysput) [reply Runtime/Model/JsonData/Annotations.cs:43]: > I don't see this bug fix in release notes.
Adding info to changelog - I focused there only on information that impacts users.
>  Also, how does one disable environment variables?
Info is available here:
https://github.com/backtrace-labs/backtrace-unity#data-privacy

File: CHANGELOG.md (hunk 1 above)
Comment (konraddysput) [reply CHANGELOG.md]: done

**Pull request merged**